src/app/components/ActionButton/ActionButton.tsx

```diff
@@ -4,7 +4,7 @@ import styles from './ActionButton.module.css';
 
 type ActionButtonProps = {
   children: ReactNode;
-  onClickHandle?: () => void;
+  onClickHandle: () => void;
```

Comment:
```suggestion
  onClick: () => void;
```
Either `onClick` or `handleClick`